tests/app.test.js

```diff
@@ -0,0 +1,152 @@
+    .then(() => client.query('INSERT INTO photos VALUES (1,1,\'https://images.unsplash.com/photo-1501088430049-71c79fa3283e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80\',\'https://images.unsplash.com/photo-1501088430049-71c79fa3283e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80\'), (2,1,\'https://images.unsplash.com/photo-1534011546717-407bced4d25c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2734&q=80\',\'https://images.unsplash.com/photo-1534011546717-407bced4d25c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80\'), (3,1,\'https://images.unsplash.com/photo-1549831243-a69a0b3d39e0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2775&q=80\',\'https://images.unsplash.com/photo-1549831243-a69a0b3d39e0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80\'), (4,1,\'https://images.unsplash.com/photo-1527522883525-97119bfce82d?ixlib=rb-1.2.1&auto=format&fit=crop&w=668&q=80\',\'https://images.unsplash.com/photo-1527522883525-97119bfce82d?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80\'), (5,1,\'https://images.unsplash.com/photo-1556648202-80e751c133da?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80\',\'https://images.unsplash.com/photo-1556648202-80e751c133da?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80\'), (6,1,\'https://images.unsplash.com/photo-1532543491484-63e29b3c1f5d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80\',\'https://images.unsplash.com/photo-1532543491484-63e29b3c1f5d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80\') ON CONFLICT (photos_id) DO NOTHING'))
+    .then(() => client.query('INSERT INTO sku VALUES (1,1,\'XS\',8), (2,1,\'S\',16), (3,1,\'M\',17), (4,1,\'L\',10), (5,1,\'XL\',15), (6,1,\'XL\',4) ON CONFLICT (sku) DO NOTHING'))
+    .then(() => client.query('INSERT INTO related_items VALUES (1,1,2), (2,1,3), (3,1,8), (4,1,7) ON CONFLICT (related_id) DO NOTHING'))
+);
+
+afterAll(done => client.end(done));
+
+describe('GET /products', () => {
+  describe('when a request is made with no params', () => {
+    test('should return 5 products by default', async () => {
+      jest.setTimeout(30000);
+      await request(app).get('/products')
+        .then(response => {
+          expect(response.body.length).toBe(5);
+        });
+    });
+    test('should return first 5 products in database by default', async () => {
+      jest.setTimeout(30000);
+      await request(app).get('/products')
+        .then(response => {
+          expect(response.body[0].name).toEqual(expect.stringContaining('Camo Onesie'));
```

Comment:
I don't understand why you do this instead of `expect(response.body[0].name).toBe('Camo Onesie')`